Add permissions and UHC docs (#516)
medic/cht-docs: content/en/apps/reference/app-settings/user-permissions.md

keywords: user-roles user-permissions
---

Permissions are defined by the `permissions` object in the `base_settings.json` file. Permissions can also be configured using the App Management app.

> Each permission is defined in terms of an array of user role identifiers that have the permission granted.

Reviewer: ```suggestion
Each permission is defined as an array of user role identifiers that have the permission granted.
```
